macros/DekiScript-Web.ejs

```diff
@@ -1,35 +0,0 @@
-<% module.exports = buildAPI({
- 
-    // Insert a hyperlink.
-    // [See also](http://developer.mindtouch.com/en/docs/DekiScript/Reference/DekiScript_Functions_and_Variables/Web.Link)
-    link: function (uri, text, title, target) {
-        var out = [
-            '<a href="' + this.spacesToUnderscores(kuma.htmlEscape(uri)) + '"'
-        ];
-        if (title) {
-            out.push(' title="' + kuma.htmlEscape(title) + '"');
-        }
-        if (target) {
-            out.push(' target="' + kuma.htmlEscape(target) + '"');
-        }
-        out.push('>', kuma.htmlEscape(text || uri), '</a>');
-        return out.join('');
-    },
-    
-    // Convert source to HTML by optionally fetching and parsing it.
-    // [See also](http://developer.mindtouch.com/en/docs/DekiScript/Reference/DekiScript_Functions_and_Variables/Web.Html)
-    html: function (str) {
```

Comment:
LOL, what a useful function! 😄 Nice catch noticing that this isn't used and not including it your new code!